Migrations/Mysql/Version20250403223549.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Flow\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Platforms\AbstractMySQLPlatform;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20250403223549 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf(
+            !$this->connection->getDatabasePlatform() instanceof AbstractMySQLPlatform,
+            "Migration can only be executed safely on '\Doctrine\DBAL\Platforms\AbstractMySQLPlatform'."
+        );
+
+        $schema->dropTable('flownative_tokenauthentication_security_model_hashandroles');
```

Comment:
Now, if the table does not exist, because this isn't an update… 💥

Reply:
You're right. Changed it

Reply:
If dropping would be the "correct solution", this would be needed:
```suggestion
        $this->addSql(<<<'SQL'
            DROP TABLE IF EXISTS flownative_tokenauthentication_security_model_hashandroles
        SQL);
```